src/Handler/StreamHandler.php

```diff
@@ -75,9 +76,9 @@ public function __invoke(RequestInterface $request, array $options): PromiseInte
     private function invokeStats(
         array $options,
         RequestInterface $request,
-        $startTime,
+        ?float $startTime,
         ResponseInterface $response = null,
-        $error = null
+        ?\Throwable $error = null
```

Comment:
Don't need the quesionmark here.

Reply:
fixed in 583f8291db35249b94ca8893068002a654caf4a1